recipes/b2/portable/conanfile.py

```diff
@@ -107,6 +108,12 @@ def _bootstrap_env(self):
             os.environ.clear()
             os.environ.update(saved_env)
 
+    def _write_project_config(self, cxx):
+        with open(os.path.join(self.source_folder, "project-config.jam"), "w") as f:
+            f.write(
+                f"using {self.options.toolset} : : {cxx} ;\n"
```

Comment:
Is this for debug or confirm it works?

Reply:
It's writing to a project config file to tell b2 where the compiler is located. I'm a b2 novice, but this seemed like the best way to configure the project to use the compiler path from the CXX env var.

Without this, the b2 install stage uses the wrong compiler path.